guvnor-ala/guvnor-ala-distribution/src/test/java/org/guvnor/ala/services/tests/RuntimeEndpointsTestIT.java

```diff
@@ -43,88 +43,57 @@
     @Ignore
     public void checkService() {
         Client client = ClientBuilder.newClient();
-        WebTarget target = client.target(APP_URL);
-        ResteasyWebTarget restEasyTarget = (ResteasyWebTarget) target;
-        RuntimeProvisioningService proxy = restEasyTarget.proxy(RuntimeProvisioningService.class);
+        WebTarget target = client.target( APP_URL );
```

Comment:
Seems like you're not using [Kie Java conventions|https://github.com/kiegroup/droolsjbpm-build-bootstrap/blob/master/ide-configuration/intellij-configuration/code-style/intellij-code-style_droolsjbpm-java-conventions.xml] for formatting :-(